project.html

```diff
@@ -135,10 +135,6 @@
   <p>
   Overall, the main point of baselines and oracles is to get you to look at the problem
   carefully and think about what's possible.
-  The accuracies of state-of-the-art systems on the dataset
```

Comment:
I think this might be nice to keep. There was some confusion about what constitutes a baseline/oracle during lecture. Concrete examples like this provide good direction.